tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/Scenario.scala

```diff
@@ -96,12 +96,22 @@ case class Scenario(categories: List[String], featureName: String, number: Optio
 
           case (ctx, RegisterProcedure(signature, table, _)) =>
             ctx.graph match {
-              case support: ProcedureSupport =>
-                support.registerProcedure(signature, table)
+              case procedureSupport: ProcedureSupport =>
+                procedureSupport.registerProcedure(signature, table)
               case _ =>
             }
             Right(ctx)
```

Comment:
To keep the code structure parallel with cvs support:

```suggestion
              case procedureSupport: ProcedureSupport =>
                Right(procedureSupport.registerProcedure(signature, table))
              case _ =>
                Right(ctx)
            }
```

Reply:
`.registerProcedure()` returns a unit, so  this is not possible, unfortunately.